src/blocks.cpp

```diff
@@ -535,9 +535,30 @@ void BlockHit(int A, bool HitDown, int whatPlayer)
 
 
     }
-    else if(b.Special >= 1000) // New spawn code
+    else if(b.Special >= 100) // New spawn code
     {
-        C = b.Special - 1000; // this finds the NPC type and puts in the variable C
+        // represents behavior for ancient 101 / 104 / 201 Special values (other ancient values replaced in OpenLevel)
+        bool is_ancient = false;
+
+        if(b.Special >= 1000)
+        {
+            C = b.Special - 1000; // this finds the NPC type and puts in the variable C
+        }
+        else if(b.Special == 201)
+        {
+            C = NPCID_LIFE_S3;
+            is_ancient = true;
+        }
+        else if(b.Special == 104)
+        {
+            C = NPCID_GRN_BOOT;
+            is_ancient = true;
+        }
+        else // b.Special == 101
+        {
+            C = NPCID_FODDER_S3;
+            is_ancient = true;
+        }
```

Comment:
This code, seems, incomplete, there are MANY ancient codes to be properly treated:

See this list as a hint: <https://github.com/WohlSoft/PGE-File-Library-STL/blob/master/file_rw_lvl.cpp#L354-L381>

Anyway, I'll try to make a demo to verify the work...

Reply:
Okay, I found the place where code gets converted, all fine.